pkg/utils/unorderedset_test.go

```diff
@@ -16,12 +17,17 @@ func TestToSlice(t *testing.T) {
 	set.Insert("two")
 	set.Insert("one")
 	expected := []string{"one", "two"}
-	actual := set.ToSlice()
+	actualSlice := set.ToSlice()
+	actual := []string{
+		actualSlice[0].(string),
+		actualSlice[1].(string),
+	}
+	sort.Strings(actual)
```

Comment:
Just a thought gomega comes with nice matcher libraries with idioms such as `Contains`. Probably would be useful here?

Reply:
Thanks for challenging me to look into Gomega + Ginkgo.  Seems really well done!